testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveDatabricksHttpMetastore.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.hive;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.tempto.BeforeMethodWithContext;
+import io.trino.tempto.ProductTest;
+import io.trino.testng.services.Flaky;
+import io.trino.tests.product.utils.DatabricksUnityCatalogUtils;
+import org.testng.annotations.Test;
+
+import java.util.Locale;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static io.trino.tests.product.TestGroups.HIVE_DATABRICKS_HTTP_METASTORE;
+import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_ISSUE;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_MATCH;
+import static io.trino.tests.product.utils.DatabricksUnityCatalogUtils.getCreateSchemaWithLocationQuery;
+import static io.trino.tests.product.utils.DatabricksUnityCatalogUtils.getRandomPathInExternalLocation;
+import static io.trino.tests.product.utils.DatabricksUnityCatalogUtils.getS3BucketName;
+import static io.trino.tests.product.utils.QueryExecutors.onDelta;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestHiveDatabricksHttpMetastore
+        extends ProductTest
+{
+    private String unityCatalogCatalogName;
+    protected String bucketName;
+
+    @BeforeMethodWithContext
+    public void setUp()
+    {
+        bucketName = requireNonNull(System.getenv("S3_BUCKET"), "Environment variable not set: S3_BUCKET");
+        unityCatalogCatalogName = DatabricksUnityCatalogUtils.getTestCatalogName();
+    }
+
```

Comment:
Please consider (if it works) adding also a test for a partitioned table.
In case of Hive, we rely on the metastore for retrieving the partitions.

Reply:
Talked offline - DML on partitioned Hive tables doesn't yet work on Unity. Disregard my comment.